public/data/Roshan-Sharma.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Roshan Sharma",
+  "type": "personal",
+  "bio": "I have completed my graduation in 2020. I have done MCA with a CGPA of 7.57/10. I am currently looking for a Software Developer role. I am open to collaborate with any open source project to increase my skills.",
+  "avatar": "https://github.com/RonenTGreat.png",
```

Comment:
You might have entered the wrong URL here! 

```suggestion
  "avatar": "https://github.com/imrosun.png",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix